deploy/efs-csi/sss-config.yaml

```diff
@@ -0,0 +1,2 @@
+matchLabels:
+  hive.openshift.io/efs-csi: "true"
```

Comment:
What is the mechanisim we expect to have enabling this feature on OSD clusters?  If driven through OCM then label should be `api.openshift.com/efs-csi` as that's the prefix used from that system.  If we never expect it to be driven through OCM then perhaps `managed.openshift.io/efs-csi`.  The `hive.openshift.io` prefix implies it is a value set by hive and it is not.

Reply:
Okay, I had no idea, just guessed based on the fact that this was going to need to be done to a hive-managed cluster.

The mechanism is "SRE labels by hand" using `oc label clusterdeployment`. See https://github.com/openshift/ops-sop/pull/744/files?short_path=c81e05c#diff-c81e05c90920950575849b3827d2fbfc

As to whether this will _ever_ be done via OCM, I don't know, but probably not. This SSS solution is really kind of an intermediate step towards having an operator to do the heavy lifting (whereupon the label will be unnecessary? or at least different?)

So, given all that, what do you recommend?

Reply:
I would recommend changing it to `managed.openshift.io/efs-csi` since this feature may not be an optional addition through OCM but a default from SRE (thinking ahead). For the moment SRE will be setting it so thats what makes sense to me.